src/app/pages/component-category-list/component-category-list.scss

```diff
@@ -22,12 +22,9 @@
   }
 }
 
-.docs-component-category-list-card-image {
-  width: 100%;
-  height: 160px;
-  background-size: contain;
-  background-repeat: no-repeat;
-  background-position: center;
+:host /deep/ .docs-component-category-list-card-image svg {
```

Comment:
I'd like to avoid `/deep/` completely, instead preferring to add a global style (or, barring that, turning off view encapsulation for the component)